src/lib.rs

```diff
@@ -30,20 +30,7 @@
 //!
 //! ```
 
-#![doc(html_root_url = "https://docs.rs/yaml-rust2/0.5.0")]
```

Comment:
Any reason you removed all attributes?

Reply:
I found this thread about `html_root_url` ~ https://users.rust-lang.org/t/you-do-not-need-html-root-url/73678

It doesn't sound like we strictly need these attributes as far as I can tell. I think we can consider this Resolved so I'll resolve this thread for now but if you think it's still useful I can restore the attributes.

Reply:
My comment only pointed to this attribute, but you removed all clippy-related attributes as well. I should have highlighted the whole block, my bad.

I'll merge this and re-enable warnings later ;)